content/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Kestra Raises $3 Million to Build the Universal Open Source Orchestration Solution for All Engineers"
+description: "Kestra raises $3 million in funding to build a universal open-source orchestration platform, empowering software and data engineers across industries."
+date: 2023-10-05T10:00:00
+category: News & Products Updates
+author:
+  name: Emmanuel Darras
+  image: "edarras"
+image: /blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.jpg
+---
+
+We are thrilled to announce that Kestra has raised **$3 million of funding** to build the leading universal open-source orchestrator. We aim to empower a wide range of users, from software developers to data engineers, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **powerful** orchestration solutions for any kind of process and data automation usage.
```

Comment:
```suggestion
We are thrilled to announce that Kestra has raised **$3 million of funding** to make orchestration more accessible, collaborative, and scalable. 

We aim to empower a wide range of users, from software developers to data engineers, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **versatile** orchestration.

Through a seamless UI and IDE access, all workflows — from those orchestrating mission-critical applications, business processes, and data pipelines to simple automations — can be version controlled and governed using engineering best practices.
```

not strong enough as intro yet -- maybe this to show the problem kestra tries to solve?

Reply:
Yep, I stick to the PR for the content as every word was chosen, but I like also being straight to the point and informative